ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TipsLayout.kt

```diff
@@ -0,0 +1,94 @@
+package org.cru.godtools.tutorial
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+//import org.cru.godtools.R
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import java.nio.file.Files.size
+
+@Preview(showBackground = true)
+@Composable
+internal fun TipsLayout() = GodToolsTheme() {
+    Column(
+        modifier = Modifier.fillMaxSize().padding(top = 100.dp),
+
+        //verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ){
+        Spacer(modifier = Modifier.height(100.dp))
+        val composition by rememberLottieComposition(
+            LottieCompositionSpec
+                .RawRes(org.cru.godtools.tutorial.R.raw.anim_tutorial_tips_people)
+        )
+        var isPlaying by remember {
+            mutableStateOf(true)
+        }
+
+// for speed
+        var speed by remember {
+            mutableStateOf(1f)
+        }
```

Comment:
same thing with `speed`, it won't be changing for the tutorial, so it can be hard-coded in `animateLottieCompositionAsState`

Reply:
the default value for `speed` is `1f`, so you wouldn't need to pass this argument to `animateLottieCompositionAsState` at all